### PR TITLE
SER-2902 Disable loan submit button for 5 seconds to avoid double clicks

### PR DESCRIPTION
**Pull request by simiyu17**
## Description
Disable loan submit button for 5 seconds to avoid double clicks

## Changes Included in PR
- Jira Ticket : [https://oneacrefund.atlassian.net/browse/SER-2902](https://oneacrefund.atlassian.net/browse/SER-2902)

## PR Checklist
- [ ] Xray tests created
- [ ] PM tested and approved
- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`

## Screenshots, if any

.

### CODE REVIEW

File: src/app/loans/loans-account-stepper/loans-account-preview-step/loans-account-preview-step.component.ts [hunk 1]
```diff
@@ -25,8 +25,19 @@ export class LoansAccountPreviewStepComponent implements OnInit {
...
+  isLoanSubmitButtonDisabled = false
+
   constructor() { }
 
   ngOnInit() { }
 
+  submitAndDisbleLoanSubmitButton(){
+    this.isLoanSubmitButtonDisabled = true;
+    setTimeout(() => {
+      this.isLoanSubmitButtonDisabled = false;
```
Comment (gilclefes) [inline src/app/loans/loans-account-stepper/loans-account-preview-step/loans-account-preview-step.component.ts:37]: Can we add value as environment variable, they should also be added to the env.js and eve.template.js so we can update them from docker.

**Review by gilclefes — APPROVED** [review]
looks good to me

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
     * [SER-2961] - Client's Date of Birth (DOB) should be available on the client profile.
     * [SER-2955] - Revise the global configuration pages to incorporate the new isStringValue flag field.
+    * [SER-2902] - Disable loan submit button for 5 seconds to avoid double clicks.
```
Comment (amydoxym) [inline RELEASE_NOTES.md:7]:
```suggestion
    * [SER-2902] - Disable loan submit button for a specific period to avoid double clicks.
```

**Review by amydoxym — CHANGES_REQUESTED** [review]
nice Implementation..just update the release notes as suggested

File: RELEASE_NOTES.md (hunk 2 above)
Comment (simiyu17) [reply RELEASE_NOTES.md]: @amydoxym  Done

**Review by amydoxym — APPROVED** [review]

**Pull request merged**